packages/@ember/application/instance.ts

```diff
@@ -61,10 +65,10 @@ const ApplicationInstance = EngineInstance.extend({
     @private
     @property {String|DOMElement} rootElement
   */
-  rootElement: null,
+  rootElement: string | Element | null = null;
```

Comment:
This is the per-instance vs. per-prototype thing again. Here, though, I think it *should* not matter: it's a single string or element (or `null`!) and I *think* there will be a per-instance version of the actual string or element in any case—there basically has to be.